cypress/support/Actions/auth/LoginActions.ts

```diff
@@ -99,4 +138,44 @@ export default class LoginActions {
       }
     );
   }
+
+  /**
+   * Logs in with SSO and caches session across specs
+   */
+  loginWithSSOAndSaveSession(
+    userCreds: Omit<UserLoginCredentialProps, 'password'>,
+    cacheAcrossSpecs = true
+  ) {
+    cy.session(
+      session.createSessionIdWithSSO(userCreds),
+      () => this.doLoginViaSSO(userCreds),
+      {
+        cacheAcrossSpecs,
+        validate: () => {
```

Comment:
This validation logic is repeated in the `loginWithSSOAndSaveSession` and `loginWithCredAndSaveSession` methods.